src/components/preview-ghost/PreviewGhost.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import * as React from 'react';
-import Ghost from '../../../../../components/ghost/Ghost';
+import Ghost from '../ghost/Ghost';
```

Comment:
Thoughts on moving to `message-preview-content` folder with the other component in this file? Unless you think they will be used standalone

Reply:
PreviewGhost is used not only in MessagePreviewContent, but also in message-center-modal/MessageCenterModal.js, that's why I created a new folder for it